### PR TITLE
Add non-HA Teleport cluster to Deploy with Helm links

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

**Review by hugoShaka — APPROVED** [review]

File: docs/pages/deploy-a-cluster/helm-deployments.mdx [hunk 1]
```diff
@@ -9,8 +9,10 @@ layout: tocless-doc
 These guides show you how to set up a full self-hosted Teleport deployment using
 our `teleport-cluster` Helm chart.
 
+- [Teleport Cluster](./helm-deployments/kubernetes-cluster.mdx): Running a non-HA Teleport cluster in a Kubernetes cluster. A
```
Comment (alexfornuto) [inline docs/pages/deploy-a-cluster/helm-deployments.mdx:12]: Can we expand "non-HA" to be more friendly to those not familiar with the jargon?

Comment (alexfornuto) [reply docs/pages/deploy-a-cluster/helm-deployments.mdx:12]: Also, this sentence lacks a subject.

File: docs/pages/deploy-a-cluster/helm-deployments.mdx [hunk 2]
```diff
@@ -9,8 +9,10 @@ layout: tocless-doc
...
+  great starting point to trying a self-hosted Teleport with minimal resources.
```
Comment (alexfornuto) [inline docs/pages/deploy-a-cluster/helm-deployments.mdx:13]: This sentence also lacks a subject.

File: docs/pages/deploy-a-cluster/helm-deployments.mdx (hunk 1 above)
Comment (stevenGravy) [reply docs/pages/deploy-a-cluster/helm-deployments.mdx]: Please see updates @alexfornuto

File: docs/pages/deploy-a-cluster/helm-deployments.mdx (hunk 2 above)
Comment (stevenGravy) [reply docs/pages/deploy-a-cluster/helm-deployments.mdx]: Updated

File: docs/pages/deploy-a-cluster/helm-deployments.mdx [hunk 3]
```diff
@@ -9,8 +9,11 @@ layout: tocless-doc
 These guides show you how to set up a full self-hosted Teleport deployment using
 our `teleport-cluster` Helm chart.
 
+- [Teleport Cluster](./helm-deployments/kubernetes-cluster.mdx): Running a Teleport cluster in a Kubernetes cluster using
```
Comment (alexfornuto) [inline docs/pages/deploy-a-cluster/helm-deployments.mdx:12]: This removes SEO un-friendly "ing" and also provides a subject for the sentence.

```suggestion
- [Teleport Cluster](./helm-deployments/kubernetes-cluster.mdx): Run a Teleport cluster in a Kubernetes cluster using
```

File: docs/pages/deploy-a-cluster/helm-deployments.mdx [hunk 4]
```diff
@@ -9,8 +9,11 @@ layout: tocless-doc
...
+  the default configuration. This deployment is a great starting point to trying a self-hosted
```
Comment (alexfornuto) [inline docs/pages/deploy-a-cluster/helm-deployments.mdx:13]:
```suggestion
  the default configuration. This deployment is a great starting point to try a self-hosted 
```

**Review by alexfornuto — APPROVED** [review]

**Pull request merged**